routing/router.go

```diff
@@ -213,6 +213,44 @@ func newRouteTuple(amt lnwire.MilliSatoshi, dest []byte) routeTuple {
 	return r
 }
 
+// edgeLocator is a struct used to identify a specific edge. The direction
+// fields takes the value of 0 or 1 and is identical in definition to the
+// channel direction flag. A value of 0 means the direction from the lower node
+// pubkey to the higher.
+type edgeLocator struct {
```

Comment:
to reduce commit size this could be introduced in a separate commit :)

Reply:
This is too granular for me and I am also not sure if it makes sense to review a data structure without knowing what it will be used for.

Reply:
It is just a personal preference of mine, as I tend to jump back and forth between commits anyway :)